terraform/environment-pipelines/iam.tf

```diff
@@ -669,7 +669,8 @@ data "aws_iam_policy_document" "postgres" {
         "lambda:GetLayerVersion"
       ]
       resources = [
-        "arn:aws:lambda:eu-west-2:763451185160:layer:python-postgres:1"
+        "arn:aws:lambda:eu-west-2:763451185160:layer:python-postgres:1",
+        "arn:aws:lambda:eu-west-2:763451185160:layer:python-requests:8"
```

Comment:
Is there any reason why we're hardcoding the version? If/when we patch these layers it'll mean needing to come back to this file and remembering to bump the number.